feature-staff/src/main/java/io/github/droidkaigi/confsched2022/feature/staff/StaffUiModel.kt

```diff
@@ -0,0 +1,33 @@
+package io.github.droidkaigi.confsched2022.feature.staff
+
+import io.github.droidkaigi.confsched2022.model.Staff
+import io.github.droidkaigi.confsched2022.ui.Result
+import kotlinx.collections.immutable.PersistentList
+
+data class StaffUiModel(
+    val staffState: StaffState,
+) {
+    sealed interface StaffState {
```

Comment:
@t-nonomura Do you think you can try to use UiLoadState introduced in #287? We could make the UiModel even simpler to use.

Reply:
I fixed it.
d53e563

Your code is very concise, thank you !

Reply:
👍